tests/test_reserver.py

```diff
@@ -22,5 +29,10 @@ def test_valid_package_invalid_credentials():
 def test_valid_package_valid_credentials():
     # test not reserved name -> correct credentials
     # uploader = PyPIUploader(test_pypi_token, test_pypi=True)
-    # uploader.upload_to_pypi(get_random_name())
+    # uploader.upload(get_random_name())
     assert True == True
+
+def test_module_conflict():
```

Comment:
@AHReccese 
Why is this test on `main_pypi` instead of `test_pypi`?

Reply:
Because this special case exists in test PyPI. The associated reserved package name is `frz` and it has a main module named `freeze` leading to this conflict in reservation, at first glance `freeze` is free but underneath it isn't.
But in `test PyPI`, the name `freeze` has been directly reserved so using `freeze` name in `test PyPI` doesn't get into this error.

Reply:
Ow!
It's okay, but I suggest finding an example like this on `test_pypi` in the future.

Reply:
I agree, I will search for it, and let me know by any chance if you find one.